Conductor/files/config.properties

```diff
@@ -47,3 +47,5 @@ conductor.app.eventProcessorThreadCount=10
 
 conductor.metrics-prometheus.enabled=true
 management.endpoints.web.exposure.include=prometheus
```

Comment:
Do we use `prometheus` anymore?

Reply:
No - I think we can remove this too but I'll do it in a separate PR